index.bs

```diff
@@ -5114,35 +5114,40 @@ a [=user-verifying platform authenticator=].
 
 1. The [=[RP]=] script runs the code snippet below.
 
-1. The user agent asks the user whether they are willing to register with the [=[RP]=] using an available [=platform authenticator=].
+    1. The user agent checks if a [=user-verifying platform authenticator=] is available.
 
-1. If the user is not willing, terminate this flow.
+    1. The [=[RP]=] asks the user if they want to create a credential with it.
```

Comment:
The way this is written, does it suggest the RP shows UI asking the user about cred creation regardless of whether iUVPAA() returns true or false?  If so, do we want to do that?

Reply:
Good catch, fixed in 07b58d51007d33dca4e620b2c3d1a2e6346d0775